0.Preprocess_Data/3.data_aggregation.ipynb

```diff
@@ -10,7 +10,7 @@
         },
```

Comment:
<p>May consider breaking this down to be more modular with less code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>